src/platform/user/profile/vap-svc/constants/addressValidationMessages.js

```diff
@@ -7,6 +7,8 @@ export const ADDRESS_VALIDATION_TYPES = Object.freeze({
   MISSING_UNIT_OVERRIDE: 'missingUnitNumberOverride',
   SHOW_SUGGESTIONS_OVERRIDE: 'showSuggestionsOverride',
   SHOW_SUGGESTIONS_NO_CONFIRMED_OVERRIDE: 'showSuggestionsNoConfirmedOverride',
+  NO_SUGGESTIONS_NO_OVERRIDE: 'noSuggestionsNoOverride',
+  SHOW_SUGGESTIONS_NO_OVERRIDE: 'showSuggestionsNoOverride',
```

Comment:
What are the different use cases for `SHOW_SUGGESTIONS_NO_CONFIRMED_OVERRIDE` and `SHOW_SUGGESTIONS_NO_OVERRIDE`, and could these be combined?

Reply:
I see in the next file, never mind!